.mergify.yml

```diff
@@ -0,0 +1,50 @@
+pull_request_rules:
+- name: automatic squash-and-merge on CI success and review
+  conditions:
+  - status-success=Travis CI - Pull Request
```

Comment:
Whoops, this should just be `ci` like in Treadle: https://github.com/chipsalliance/treadle/blob/8477a3ca0cc5a264c3a0fe2f73532991e0c44845/.mergify.yml#L4